sql/table_test.go

```diff
@@ -170,8 +188,21 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		if err != nil {
 			t.Fatalf("%d: %v", i, err)
 		}
-		if !reflect.DeepEqual(d.index, schema.Indexes[0]) {
-			t.Fatalf("%d: expected %+v, but got %+v", i, d.index, schema.Indexes[0])
+		if !reflect.DeepEqual(d.primary, schema.PrimaryIndex) {
+			t.Fatalf("%d: expected %+v, but got %+v", i, d.primary, schema.PrimaryIndex)
+		}
+		if len(d.indexes) > 0 && !reflect.DeepEqual(d.indexes, schema.Indexes) {
+			t.Fatalf("%d: expected %+v, but got %+v", i, d.indexes, schema.Indexes)
 		}
+
+	}
+}
+
+func TestErrPrimaryKeyUnspecified(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	stmt, _ := parser.Parse("CREATE TABLE test (a INT, b INT, CONSTRAINT c UNIQUE (b))")
```

Comment:
yeah I did that on purpose. I feel we could benefit from reducing the size of the testing code by eliminating error checks that we know will not get triggered. Do you think we should follow this philosophy? I just feel like we really shouldn't need to be checking for errors in the testing code.